crates/uv/src/commands/project/run.rs

```diff
@@ -1016,7 +1016,15 @@ pub(crate) async fn run(
             .as_ref()
             .map(PythonEnvironment::scripts)
             .into_iter()
+            .chain(
+                ephemeral_env
+                    .as_ref()
+                    .and_then(|env| env.python_executable().parent())
+                    .into_iter(),
+            )
             .chain(std::iter::once(base_interpreter.scripts()))
+            .chain(base_interpreter.sys_executable().parent().into_iter())
```

Comment:
I'm sort of hesitant to do this so broadly, couldn't this add something like `/usr/bin` to the `PATH`? I would find that surprising. That example is a bit contrived, but it could be an arbitrary directory containing a link to the interpreter right? Should we just be special-casing the Python executable (e.g., by invoking the full path to the interpreter) instead of adding its parent directory to the `PATH`?

Reply:
Ah sorry that suggestion doesn't resolve the linked issue, in which you'd expect the executable to be on the `PATH` in a child shell. Hm.

Reply:
Another possibility is #5839

Reply:
Hmm, I don't think that's sufficient. There are other executables in the top-level too, right? I think this is just correct, at least on Windows. We could limit it to Windows?

Reply:
We could also choose to limit this to Windows, non-virtual environments.

Reply:
I'm more comfortable limiting this to Windows.